packager/packager.cc

```diff
@@ -1552,6 +1643,8 @@ void validateLayering(const core::Context &ctx, const Import &i) {
     ENFORCE(packageDB.getPackageForFile(ctx, ctx.file).exists())
     auto &thisPkg = PackageInfoImpl::from(packageDB.getPackageForFile(ctx, ctx.file));
     auto &otherPkg = PackageInfoImpl::from(packageDB.getPackageInfo(i.name.mangledName));
+    ENFORCE(thisPkg.sccID.has_value());
+    ENFORCE(otherPkg.sccID.has_value());
```

Comment:
Is it worth mentioning why we assume these to hold?

something like "expected to have run tarjans already"